client/app/bundles/course/assessment/submission/components/answers/index.tsx

```diff
@@ -17,10 +18,13 @@ import AnswerHeader from './AnswerHeader';
 import { AnswerPropsMap } from './types';
 import { QuestionHistory } from '../../reducers/history/types';
 import { updateAnswerFlagSavingStatus } from '../../reducers/answerFlags';
+import translations from '../../translations';
+import { AttachmentErrorType } from '../../pages/SubmissionEditIndex/AttachmentErrorType';
 
 interface SubmissionAnswerProps<T extends keyof typeof QuestionType> {
   answerId: number;
   graderView: boolean;
+  error: AttachmentErrorType;
```

Comment:
alright. noted on this, I have changed it into `ErrorType` instead.
In the future, should there be any other errors supported, it can be just put inside this type